pkg/secrets/handlers/secrets_handler_k8s_test.go

```diff
@@ -29,4 +30,35 @@ func TestSecretsHandlerK8sUseCase(t *testing.T) {
 			So(err.Error(), ShouldEqual, "error map should not be empty")
 		})
 	})
+
+	Convey("updateK8sSecretsMapWithConjurSecrets", t, func () {
+		secret := []byte{'s', 'u', 'p', 'e', 'r'}
+		conjurSecrets := make(map[string][]byte)
+		conjurSecrets["account:variable:allowed/username"] = secret
+
+		newDataEntriesMap := make(map[string][]byte)
+		newDataEntriesMap["username"] = []byte("allowed/username")
+
+		k8sSecretsMap := make(map[string]map[string][]byte)
+		k8sSecretsMap["mysecret"] = newDataEntriesMap
+
+		pathMap := make(map[string]string)
+		pathMap["allowed/username"] = "mysecret:username"
+
+		k8sSecretsStruct := k8s.K8sSecretsMap{k8sSecretsMap, pathMap}
+		err := updateK8sSecretsMapWithConjurSecrets(&k8sSecretsStruct, conjurSecrets)
+
+		eq := reflect.DeepEqual(k8sSecretsStruct.K8sSecrets["mysecret"]["username"], secret)
+		Convey("Returns true if K8sSecretsMap's variable ID is replaced with corresponding secret value", func() {
```

Comment:
the message here is not what we're looking for. We need to write here the expected behaviour. For example:
`Convey("Replaces the Variable IDs in k8sSecretsMap to their corresponding secret values without errors"...`

This is actually a mistake that nessi did in his UTs (you can see how he got the idea from `authenticator_test.go`) so please go over all the `Convey`s in our code and change the text to be self-explanatory. This will also help you understand what we test in each UT.